### PR TITLE
feat(create): add checkpoints for hands on

### PR DESCRIPTION
**Pull request by Permas-Teo**

### CODE REVIEW

File: frontend/src/screens/Todo.tsx [hunk 1]
```diff
@@ -90,8 +90,7 @@ function Todo() {
         </Table>
       </Form>
       <Button
-        size="sm"
-        variant="primary"
+        // Checkpoint 1: Modify Button's `size` and `variant` props
```
Comment (jasxlee) [inline frontend/src/screens/Todo.tsx:94]: Just wondering, do we want to include a comment where they can find what kind of props are passable? E.g.: 
// Refer to docs: https://react.designsystem.tech.gov.sg/iframe.html?viewMode=docs&id=components-button--button-with-icon 
Not sure if that will be more helpful or too much handholding haha

File: frontend/src/components/TodoItem.tsx [hunk 2]
```diff
@@ -15,59 +15,46 @@ export type TodoItemProps = {
...
+  // Checkpoint 3: Implement a method like "updateTodoItem" to delete a TodoItem
 
-  const generateGptContent = useCallback(async (id: string) => {
-    setIsLoading(true);
-    await axios.post(`${CONFIG.API_ENDPOINT}/todos/${id}/generate`);
-    setIsLoading(false)
-    props.refreshToDos();
-  }, []);
+  // Checkpoint 4: Implement a method like "updateTodoItem" to generate the ChatGPT content
+  // Considering how long it takes to generate the content, how could using `setIsLoading` be helpful?
```
Comment (jasxlee) [inline frontend/src/components/TodoItem.tsx]: Just a suggestion: do you think we could split this into checkpoint 4.1 and 4.2? 
My thought process for the reasoning behind splitting is: Once they create the 'generateGptContent' function, we can get them to observe how there's no UI indication that the function is actually working at the backend. Then from here we can let them better understand the rationale of having isLoading. What are your thoughts?

File: frontend/src/screens/Todo.tsx (hunk 1 above)
Comment (Permas-Teo) [reply frontend/src/screens/Todo.tsx:94]: sgtm! there could be different places to look for documentation, good to set context on how to self help

File: frontend/src/components/TodoItem.tsx (hunk 2 above)
Comment (Permas-Teo) [reply frontend/src/components/TodoItem.tsx]: sgtm!

**Pull request merged**